### ISSUE TITLE
cl : Command line error D8021 : invalid numeric argument '/Wextra'

### ISSUE THREAD

**Issue by tonysepia** [jonashaag/bjoern#135]
When running 'pip install bjoern' on Windows 10:

"cl : Command line error D8021 : invalid numeric argument '/Wextra'
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\VC\\Tools\\MSVC\\14.13.26128\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2"

Apparently the compile flag -Wextra is meant for GCC and we are trying to use that on cl (the Microsoft compiler) too

**Comment by jonashaag** [comment]
PR welcome

**Comment by tonysepia** [comment]
#136 added a PR

**Comment by vforvoid** [comment]
@jonashaag hi there. Did you have time to check this PR?

**Comment by tonysepia** [comment]
@vforvoid I don't know the product too well, but even after fixing this on my side I suspect that it will be very difficult to compile the code on Windows. Basically, I have immediately encountered another error after resolving one with 'unfamiliar' compiler flags. Let's see if @jonashaag intends to make his creation Windows-friendly ;)

**Comment by jonashaag** [comment]
No, not written with Windows in mind. PR for complete Windows support welcome, but I'm not going to officially support Windows
